docker-compose.yml

```diff
@@ -9,4 +9,4 @@ services:
     depends_on:
       - emulator
   emulator:
-    image: gcr.io/cloud-spanner-emulator/emulator
+    image: gcr.io/cloud-spanner-emulator/emulator:1.4.9
```

Comment:
Fix spanner-emulator version to 1.4.9 until issue is fixed. 
https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/issues/98